src/python/pants/releases/BUILD

```diff
@@ -11,5 +11,9 @@ python_binary(
   tags = {"partially_type_checked"},
 )
 
-# Note that packages.py is not part of a target, it should be run with pex.
-# Its dependencies and such can be found in build-support/bin/release.sh.
+# While this is a target, it should be run with pex. See build-support/bin/release.sh.
```

Comment:
Unrelated, but: it's not really clear why... we should always have a copy of pants available to run it with.

Reply:
Not sure. I'll be touching this code a lot the next few days, so might change this. We already require Pants to run `generate_travis_yml.py`.